extension/src/shared/plateau/featureInspector/attributes.ts

```diff
@@ -0,0 +1,327 @@
+import { get } from "lodash-es";
```

Comment:
This is derived from [reearth-plateauview](https://github.com/eukarya-inc/reearth-plateauview/blob/55deed218905031edf5f47041359f44585a4aca6/plugin/web/extensions/infobox/core/utils/attributes.ts)